### PR TITLE
Improves relative paths for react-router

### PR DESCRIPTION
**Pull request by saiberz**
Improves relative paths for react-router

### CODE REVIEW

**Comment by saiberz** [comment]
captura de pantalla 2015-09-10 a las 12 48 05

**Comment by jeffdm** [comment]
Thanks! Awesome!

One last thing, can you sign your commit?

```
$ git commit --amend -s --no-edit
$ git push -f
```

**Comment by saiberz** [comment]
Done :)

File: src/routes.js [hunk 1]
```diff
@@ -32,24 +32,24 @@ var App = React.createClass({
...
         </Route>
       </Route>
-      <Route name="new" path="containers/new">
+      <Route name="new" path="new">
         <DefaultRoute name="search" handler={NewContainerSearch}/>
-        <Route name="pull" path="containers/new/pull" handler={NewContainerPull}></Route>
+        <Route name="pull" path="pull" handler={NewContainerPull}></Route>
       </Route>
       <Route name="preferences" path="/preferences" handler={Preferences}/>
       <Route name="about" path="/about" handler={About}/>
```
Comment (FrenchBen) [inline src/routes.js]: Shouldn't the above be cleaned up as well (no root '/' since part of the parent)?

**Comment by FrenchBen** [comment]
Other than my small comment LGTM :+1:

File: src/routes.js (hunk 1 above)
Comment (saiberz) [inline src/routes.js]: Agree, maybe both should have their own handler as well as account has or is it right inside containers?

Comment (FrenchBen) [inline src/routes.js]: I think removing the initial '/' from their path is good for now and matches the work done above. They already are outside of containers, which is how it should be.

Comment (saiberz) [inline src/routes.js]: :+1:

**Comment by jeffdm** [comment]
LGTM

**Pull request merged**